File: tfh-backend/src/main/java/com/example/tfhbackend/dto/TableDTO.java
package com.example.tfhbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Data
public class TableDTO {
    private Long id;
    private Integer ordinalNumber;
    private Integer nrOfSpots;
}

File: tfh-mobile/app/src/main/java/com/example/tfhmobile/dto/TableDTO.kt
package com.example.tfhmobile.dto

class TableDTO constructor(private val id: Long,
                           private val ordinalNumber: Int,
                           private val nrOfSPots: Int,
                           private val currentBookingId: Long?) {
    fun getId(): Long {
        return id
    }

    fun getOrdinalNumber(): Int {
        return ordinalNumber
    }

    fun getNrOfSpots(): Int {
        return nrOfSPots
    }

    fun getCurrentBookingId() : Long? = currentBookingId;
}